content/c-sharp/concepts/math-functions/terms/max/max.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'Math.Max()'
+Description: 'Returns the larger of two specified numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Max()`** method returns the larger of two specified numbers.
+
+## Syntax
+
+```pseudo
+Math.Max(num1, num2)
+```
+`Math.Max()` returns the greater value from `num1` and `num2`.
+
+## Example
+
+The following example compares two integers `i1` and `i2` and writes the greater integer to the console.
+
+```cs
+using System;
+
+class Program {
```

Comment:
```suggestion
public class Example {
```

Reply:
I agree with this change, was thinking about it too.